wormhole-connect/src/config/index.ts

```diff
@@ -19,7 +19,32 @@ const el = document.getElementById('wormhole-connect');
 if (!el)
   throw new Error('must specify an anchor element with id wormhole-connect');
 const configJson = el.getAttribute('config');
-export const config: WormholeConnectConfig = JSON.parse(configJson!) || {};
+export const config: WormholeConnectConfig = JSON.parse(configJson!) || {
+  moreTokens: {
+    label: 'More Tokens',
+    href: 'portalbridge.com',
+  },
+  extraNetworks: {
+    label: 'Extra Networks',
+    href: 'portalbridge.com',
+    description:
+      'You will leave this site and be redirected to the Portal Token Bridge to continue your operation.',
+    networks: [
+      {
+        name: 'algorand',
+        icon: "data:image/svg+xml,%3C%3Fxml version='1.0' encoding='utf-8'%3F%3E%3Csvg viewBox='532.3494 45.7309 238.36 238.72' width='238.36' height='238.72' xmlns='http://www.w3.org/2000/svg'%3E%3Cg id='lINT7W' transform='matrix(1, 0, 0, 1, 326.5293884277344, -159.9090576171875)'%3E%3Cpolygon class='cls-1' points='444.18 444.32 406.81 444.32 382.54 354.04 330.36 444.33 288.64 444.33 369.29 304.57 356.31 256.05 247.56 444.36 205.82 444.36 343.64 205.64 380.18 205.64 396.18 264.95 433.88 264.95 408.14 309.71 444.18 444.32' style='fill: rgb(255, 255, 255);'/%3E%3C/g%3E%3C/svg%3E",
```

Comment:
nit: Can we put these svgs into their own files under the icons directory and import them here?

Reply:
if we go with my comment above, we wouldn't have this here at all 🙂

Reply:
removed